sources/models/src/lib.rs

```diff
@@ -140,6 +140,14 @@ struct ContainerImage {
     superpowered: bool,
 }
 
+// Network settings. These settings will affect host service components' network behavior
+#[model]
+struct NetworkSettings {
+    http_proxy: Url,
+    https_proxy: Url,
```

Comment:
These don't need to be optional?

Reply:
They don't. With how the template is written right now, `HTTP_PROXY` and `HTTPS_PROXY` are only rendered if their corresponding settings are set.

Reply:
(The `#[model]` annotation makes all fields `Option`al.)